#!/usr/bin/env python
PACKAGE = "cv_camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("cv_cap_prop_gain", double_t, 0, "Gain", -10, 0, 10)
gen.add("cv_cap_prop_contrast", double_t, 0, "Contrast", -10, 0, 10)
gen.add("cv_cap_prop_brightness", double_t, 0, "Brightness", -10, 0, 10)
gen.add("cv_cap_prop_exposure", double_t, 0, "Exposure", -10, 0, 10)

exit(gen.generate(PACKAGE, "cv_camera", "CvCamera"))
